src/NUnitTestAdapter/AdapterSettings.cs

```diff
@@ -301,6 +315,20 @@ public void Load(string settingsXml)
 
             // Update NumberOfTestWorkers based on the DisableParallelization and NumberOfTestWorkers from runsettings.
             UpdateNumberOfTestWorkers();
+
+
+            string ValidatedPath(string path, string what)
+            {
+                try
+                {
+                    return Path.GetFullPath(path);
+                }
+                catch (Exception)
```

Comment:
`(Exception)` is redundant since the runtime wraps anything thrown into an Exception object.

Reply:
Not according to R#

Reply:
You're right here too of course, but then I should change the dotsettings file.  I'll have a look at do that if I don't see any side effects.